apps/charterafrica/src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -67,20 +80,50 @@ const OpportunityCard = React.forwardRef(function OpportunityCard(props, ref) {
           >
             {title}
           </Typography>
-          <Typography color="neutral.dark" variant="p1">
-            {description}
-          </Typography>
-          <Typography
+          <RichText
             color="neutral.dark"
-            variant="caption"
-            sx={{
-              textTransform: "uppercase",
-            }}
-          >
-            {config.deadlineText}: {deadline}
-          </Typography>
+            lineClamp="3"
+            variant="p1"
+            elements={excerpt}
```

Comment:
Also add `height` or `maxHeight` style

Reply:
@koechkevin Why add heigth? isn't it handled by the variant?